src/filters/fs.rs

```diff
@@ -135,12 +135,19 @@ fn sanitize_path(base: impl AsRef<Path>, tail: &str) -> Result<PathBuf, Rejectio
     Ok(buf)
 }
 
-#[derive(Debug)]
-struct Conditionals {
-    if_modified_since: Option<IfModifiedSince>,
-    if_unmodified_since: Option<IfUnmodifiedSince>,
-    if_range: Option<IfRange>,
-    range: Option<Range>,
+/// Conditionals that define certain aspects of file serving. These are usually based on the headers of a request, though they can be manually constructed.
+#[derive(Debug, Default)]
+pub struct Conditionals {
+    /// Only respond with the file if it has been modified since this date. [See MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Modified-Since).
+    pub if_modified_since: Option<IfModifiedSince>,
```

Comment:
I would keep the fields private. That way we can add more, or change them. We also haven't exposed the `headers` crate publicly.